spec/controllers/ctc/portal/portal_controller_spec.rb

```diff
@@ -104,6 +104,21 @@
           expect(response).to redirect_to Ctc::Portal::PortalController.to_path_helper(action: :edit_info)
         end
       end
+
+      context "when the client has resubmitted 20 times" do
+        let!(:submissions) { create_list :efile_submission, 20, :rejected, tax_return: client.tax_returns.first }
+
+        before do
+          client.tax_returns.first.update(efile_submissions: submissions)
```

Comment:
Usually, it's possible to avoid writing this kind of `before` block by carefully selecting what you put into the factory. If it's unavoidable that's fine, but I strongly encourage you to figure out how to remove it, just from a code cleanliness perspective. Not a blocker.